.github/workflows/build.yaml

```diff
@@ -37,3 +37,34 @@ jobs:
           flags: unittests
           name: codecov-umbrella
           verbose: true
+  windows-tests:
+    runs-on: windows-latest
```

Comment:
I tend to avoid `*-latest` things because our builds will break when GitHub decides to update.  Pin to the latest version explicitly and have something like renovatebot update when needed is better.